docs/content/faq.md

```diff
@@ -11,22 +11,14 @@ order: 5
 
 ## Can I export my slides for use elsewhere?
 
-Yes! Appending your presentation URL with `?export` will allow you to export your presentation by flattening out your presentation so that you can Print to PDF directly from your browser. 🎉
+Yes - you can export your slides in PDF format. Appending your presentation URL with `?export` will allow you to export your presentation by flattening out your presentation so that you can Print to PDF directly from your browser. 🎉
 
 If you want a black & white version of your slides printed to PDF, append your URL with `?export&print` to get a printer-friendly, flattened, black & white print out of your slideshow.
 
-For more info about the query parameters Spectacle supports, please check out our section about it in the [Advanced Concepts doc](./advanced-concepts.md/#query-parameters).
+For more info about the query parameters Spectacle supports, please check out our section about it in the [advanced concepts documentation](./advanced-concepts.md/#query-parameters).
```

Comment:
This link seems wrong. When I'm doing `yarn start` and viewing http://127.0.0.1:3000/docs/faq and I click on this link as `http://127.0.0.1:3000/docs/faq/advanced-concepts.md/#query-parameters` I get `Oh-no! Something’s gone wrong! ... TypeError: Cannot read property 'scrollIntoView' of undefined`.

The full link when resolved should become: `http://127.0.0.1:3000/docs/advanced-concepts/#query-parameters`.